crates/common/tedge_config/src/system_services/log_config.rs

```diff
@@ -28,22 +34,44 @@ pub fn log_init(
         .log_level
         .or(flags.debug.then_some(tracing::Level::DEBUG));
 
-    if let Some(log_level) = log_level {
-        subscriber.with_max_level(log_level).init();
-        return Ok(());
-    }
-
-    if std::env::var("RUST_LOG").is_ok() {
-        subscriber
-            .with_env_filter(tracing_subscriber::EnvFilter::from_default_env())
+    let log_layer = if let Some(log_level) = log_level {
+        log_layer
+            .with_filter(LevelFilter::from_level(log_level))
+            .boxed()
+    } else if std::env::var("RUST_LOG").is_ok() {
+        log_layer
             .with_file(true)
             .with_line_number(true)
-            .init();
-        return Ok(());
-    }
-
-    let log_level = get_log_level(sname, config_dir)?;
-    subscriber.with_max_level(log_level).init();
+            .with_filter(tracing_subscriber::EnvFilter::from_default_env())
+            .boxed()
+    } else {
+        let log_level = get_log_level(sname, config_dir)?;
+        log_layer
+            .with_filter(LevelFilter::from_level(log_level))
+            .boxed()
+    };
+
+    // Audit journal
+    let audit_appender = tracing_appender::rolling::hourly("/tmp", "tedge.audit.log");
```

Comment:
This will have to be configurable ;-)

Reply:
How about changes to c8y supported operations? Would show when supported commands are added/deleted and when the set is reloaded. Recently when live-debugging with a user we were confused why `c8y_Command` was not present, so it would be ideal to be able to go into the log audit to see why it was removed!